controllers/operator/migration/migration.go

```diff
@@ -478,6 +478,67 @@ func insertIdpConfigs(ctx context.Context, mongodb *MongoDB, postgres *PostgresD
 	return
 }
 
+func insertDirectoriesAsIdpConfigs(ctx context.Context, mongodb *MongoDB, postgres *PostgresDB) (err error) {
```

Comment:
@rwhundley ,  looks like you missed adding `insertDirectoriesAsIdpConfigs` function to `copyFuncs` array